docs/statistics/hypothesis/Confidence.md

```diff
@@ -1,4 +1,115 @@
+- **Sample Standard Deviation (s):** 10 cm
+
+### Calculating Confidence Intervals
+
+Using a 95% confidence level, both studies will calculate their respective confidence intervals.
+
+- **Study A:** Due to the smaller sample size, the confidence interval will be wider, reflecting greater uncertainty.
+- **Study B:** The larger sample size results in a narrower confidence interval, indicating higher precision in the estimate.
+
+This example demonstrates that larger sample sizes lead to more precise estimates of the population mean.
+
+## The Confidence Interval Formula
+
+The confidence interval for a population mean is typically calculated using the following formula:
+
+\[
+\text{Confidence Interval} = \bar{x} \pm t^* \left( \frac{s}{\sqrt{n}} \right)
+\]
+
+### Components:
+
+- **\(\bar{x}\):** Sample mean
+- **\(t^*\):** Critical value from the t-distribution corresponding to the desired confidence level and degrees of freedom (\(n - 1\))
+- **\(s\):** Sample standard deviation
+- **\(n\):** Sample size
+
+### Steps to Calculate:
+
+1. **Determine the Sample Mean (\(\bar{x}\)):** Calculate the average from your sample data.
+2. **Find the Critical Value (\(t^*\)):** Based on the confidence level and degrees of freedom, use a t-table or statistical software.
+3. **Calculate the Standard Error (\(\frac{s}{\sqrt{n}}\)):** Measures the variability of the sample mean.
+4. **Compute the Margin of Error:** Multiply the critical value by the standard error.
+5. **Establish the Confidence Interval:** Add and subtract the margin of error from the sample mean.
+
+### Example Calculation:
+
+For **Study A** with \(n = 30\), \( \bar{x} = 170 \) cm, \( s = 10 \) cm, and a 95% confidence level:
+
+1. **Degrees of Freedom:** \(30 - 1 = 29\)
+2. **Critical Value (\(t^*\)):** Approximately 2.045 (from t-tables)
+3. **Standard Error:** \( \frac{10}{\sqrt{30}} \approx 1.825 \)
+4. **Margin of Error:** \(2.045 \times 1.825 \approx 3.727\)
+5. **Confidence Interval:** \(170 \pm 3.727 \) → [166.273 cm, 173.727 cm]
+
+## Factors Affecting Confidence Interval Width
+
+Several factors influence the width of a confidence interval:
+
+1. **Sample Size (n):**
+      - **Larger Sample Size:** Leads to a smaller standard error, resulting in a narrower confidence interval.
+      - **Smaller Sample Size:** Increases the standard error, leading to a wider confidence interval.
+
+2. **Variability in Data (s):**
+      - **Lower Variability:** Decreases the standard error, narrowing the confidence interval.
+      - **Higher Variability:** Increases the standard error, widening the confidence interval.
+
+3. **Confidence Level:**
+      - **Higher Confidence Level (e.g., 99%):** Requires a larger critical value, resulting in a wider interval.
+      - **Lower Confidence Level (e.g., 90%):** Uses a smaller critical value, leading to a narrower interval.
+
+### Visualization
+
+![Confidence Interval Width](https://i.imgur.com/ConfidenceIntervalWidth.png)
```

Comment:
Unfortunately, the picture doesn't exist anymore. I wasn't able to find a fitting illustration. Do you happen to have another one? 🙈